packages/cycle-scripts/package.json

```diff
@@ -30,12 +30,16 @@
     "cycle-scripts": "./index.js"
   },
   "dependencies": {
-    "babel-core": "^6.24.1",
```

Comment:
I very much look forward to that day of greenkeeper supporting nested packages :)